src/components/Starred.jsx

```diff
@@ -0,0 +1,42 @@
+import { useSelector, useDispatch } from 'react-redux'
+import { Link } from 'react-router-dom'
+import starredSlice from '../data/starredSlice'
+import Movie from './Movie'
+import '../styles/starred.scss'
+
+const Starred = ({viewTrailer}) => {
+
+    const state = useSelector((state) => state)
+    const { starred } = state
```

Comment:
Get starred data without creating an unnecesary variable. Should look like this:
`const { starred } = useSelector((state) => state)`